server/.golangci.yml

```diff
@@ -62,7 +62,6 @@ issues:
         channels/api4/apitestlib.go|\
         channels/api4/bot_test.go|\
         channels/api4/brand.go|\
-        channels/api4/channel.go|\
```

Comment:
Curiosity question, what's the reasoning of removing this?

Reply:
That list contains all exceptions for the `errcheck` linter. This PR fixed the `errcheck` issues in the `channels/api4/channel.go` file and hence, we can remove the exception.